mod/calculated_metrics/lib/formula/calculator/input_item/options/company_option/company_search/abstract_condition.rb

```diff
@@ -14,62 +14,75 @@ def initialize string, id
                 @table_id = id
               end
 
-              def querify str
-                str.sub(@original, sql)
+              def validate
+                validate_metric
               end
 
-              def sql
+              def where_sql
                 raise "override me"
               end
 
               def subject_sql
-                @subject_sql ||= tablefy "#{subject}_company_id"
+                @subject_sql ||= tablefy "subject_company_id"
               end
 
               def object_sql
-                @object_sql ||= tablefy "#{object}_company_id"
+                @object_sql ||= tablefy "object_company_id"
               end
 
               def metric_sql
                 tablefy "metric_id = #{@metric_id}"
               end
 
               def inverse?
-                @inverse
+                false
```

Comment:
why?

Reply:
oh, I think it's because this lookup always happens on the direct.  maybe a short comment?

Reply:
there is an extra module for inverse relationships that overrides that method